app/src/main/java/space/taran/arknavigator/mvp/view/dialog/FolderPickerDialogView.kt

```diff
@@ -19,4 +19,6 @@ interface FolderPickerDialogView : MvpView {
     fun notifyPathPicked(path: Path, rootNotFavorite: Boolean)
     @StateStrategyType(SkipStrategy::class)
     fun toastFileChosenAsRoot()
+
+    fun showToast(msg: String)
```

Comment:
@mdrlzy is this alright? Do we need `@StateStrategyType`?

Reply:
Yes, we need SkipStrategy